frontend/src/views/camp/Dashboard.vue

```diff
@@ -226,6 +240,10 @@ export default {
     periods() {
       return keyBy(this.camp().periods().items, '_meta.self')
     },
+    progressLabels() {
+      const labels = sortBy(this.camp().progressLabels().items, (l) => l.position)
+      return keyBy(labels, '_meta.self')
```

Comment:
A select option "Activities without status" could be useful. But we can do that in a later PR.

Reply:
I agree. I think we could even remove the "In Bearbeitung" and add a filter option "Kein Status"